omod/src/main/webapp/resources/scripts/navigator/navigator.js

```diff
@@ -25,74 +25,82 @@ var Navigator = {isReady: false}
 
 
 // TODO figure out out to set up mocking to test this
+// TODO this really isn't just the "keyboard controller" anymore as it adds buttons and supports mouse clickers
 function KeyboardController(formElement) {
 
     var initFormModels = function(formEl) {
         var formElement = formEl;
         if (!formElement) {
             formElement = $('div#content > form').first();
         }
+
         formElement.prepend('<ul id="formBreadcrumb" class="options"></ul>');
+        formElement.append('<div id="nav-buttons"><button id="prev-button" type="button" class="confirm" style="display:none"><icon class="fas fa-chevron-left"/></button><button id="next-button" class="confirm right" type="button"><icon class="fas fa-chevron-right"/></button></div>');
+
         var breadcrumb = formElement.find('#formBreadcrumb').first();
+        var navButtons = formElement.find('#nav-buttons').first();
 
         var sections = _.map(formElement.find('section'), function(s) {
             return new SectionModel(s, breadcrumb);
         });
 
-        var confirmationSection = new ConfirmationSectionModel($('#confirmation'), breadcrumb, _.clone(sections), formElement.hasClass('skip-confirmation-section'));
+        var confirmationSection = new ConfirmationSectionModel($('#confirmation'), breadcrumb, _.clone(sections), formElement.hasClass('skip-confirmation-section'), navButtons);
         sections.push(confirmationSection);
 
         var questions = _.flatten( _.map(sections, function(s) { return s.questions; }), true);
         var fields = _.flatten(_.map(questions, function(q) { return q.fields; }), true);
         return [sections, questions, fields];
 
-    }
-
-    var initKeyboardHandlersChain = function(questions, fields) {
-        var questionsHandler = QuestionsKeyboardHandler(questions);
-        var fieldsHandler = FieldsKeyboardHandler(fields, questionsHandler);
-        return fieldsHandler;
-    }
-
-    var initMouseHandlers = function(sections, questions, fields) {
-        sectionsMouseHandlerInitializer(sections);
-        questionsMouseHandlerInitializer(questions);
-        fieldsMouseHandlerInitializer(fields);
-    }
+    };
 
     var modelsList = initFormModels(formElement);
+
     var sections=modelsList[0], questions=modelsList[1], fields=modelsList[2];
-    initMouseHandlers(sections, questions, fields);
-    var handlerChainRoot = initKeyboardHandlersChain(questions, fields);
+    var prevButton = formElement.find('#prev-button').first();
+
+    sectionsMouseHandlerInitializer(sections);
+    questionsMouseHandlerInitializer(questions);
+    fieldsMouseHandlerInitializer(fields);
```

Comment:
I initially pulled these out of a function because it was almost making the flow harder to read.

Reply:
Yeah this is definitely an improvement